app/views/locations/index.json.jbuilder

```diff
@@ -9,5 +9,6 @@ json.features @locations do |location|
     json.booking_location_id location.booking_location&.uid.to_s
     json.phone location.phone.to_s
     json.hours location.hours.to_s
+    json.twilio_number location.twilio_number.to_s
```

Comment:
`twilio_number` is already a string so the `to_s` is superfluous in this context.